ui/imports/shared/panels/ProfileBioSocialsPanel.qml

```diff
@@ -45,10 +45,13 @@ Control {
                 let links = JSON.parse(root.userSocialLinksJson)
                 for (let i=0; i<links.length; i++) {
                     let obj = links[i]
+                    const url = obj.url
+                    const type = textToType(obj.text)
                     socialLinksModel.append({
-                                                "text": obj.text,
-                                                "url": obj.url,
-                                                "linkType": textToType(obj.text)
+                                                "text": Utils.stripSocialLinkPrefix(url, type),
+                                                "url": url,
+                                                "linkType": type,
+                                                "icon": obj.icon
```

Comment:
nit: I don't think you need the `"` around the keys

Reply:
Yeah, matter of habit, and also pre-existing code